src/extra_files.c

```diff
@@ -148,9 +150,23 @@ int walk_tree(struct manifest *manifest, const char *start)
 			if (F[i].dir) { /* Start of new dir to skip */
 				skip_dir = F[i].filename;
 				skip_len = strlen(skip_dir);
-				printf("%s/\n", F[i].filename);
+				if (fix) {
+					string_or_die(&temp, "%s%s", path_prefix, F[i].filename);
+					printf("REMOVING DIR %s/\n", F[i].filename);
```

Comment:
Is this an error? It's not an error in execution of the code/swupd, but it can be considered an error of the system state. Is that appropriate to print to stderr?